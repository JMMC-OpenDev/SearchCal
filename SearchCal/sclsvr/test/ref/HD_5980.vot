version https://git-lfs.github.com/spec/v1
oid sha256:3e35cab0397f254d23c495add3bfc6a9b7a6c9971ee8b00f952959ee420f8fb0
size 527326
