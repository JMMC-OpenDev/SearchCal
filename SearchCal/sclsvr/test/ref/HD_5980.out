version https://git-lfs.github.com/spec/v1
oid sha256:d8c4e83bdc7f0d7ce0796818b23f53dd3f962285ac2d4220c2946039b504691c
size 859415
